public-cdn-domains.txt

```diff
@@ -6,8 +6,12 @@ files:*.is-fi.re
 files:*.is-serious.business
 files:*.is-pretty.cool
 files:*.is-pretty.sexy
+files:*.is-bad.com
+files:*.is-a-bad-waifu.com
+files:*.is-a-good-waifu.com
 
 files:owo.whats-th.is
+files:owo.solutions
```

Comment:
`owo.whats-th.is` and `awau.moe` circumvent the alphabetical ordering thing, move this down to be where it should be in relation to the other `files:` domains.